sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/RowToPubsubMessage.java

```diff
@@ -40,19 +39,14 @@
  * whether config.getValue("timestampAttributeKey") is set.
  */
 class RowToPubsubMessage extends PTransform<PCollection<Row>, PCollection<PubsubMessage>> {
-  private final Row config;
   private final Boolean useTimestampAttribute;
 
-  private RowToPubsubMessage(Row config, Boolean useFlatSchema, Boolean useTimestampAttribute) {
-    checkArgument(useFlatSchema, "RowToPubsubMessage is only supported for flattened schemas.");
-
-    this.config = config;
+  private RowToPubsubMessage(Boolean useTimestampAttribute) {
```

Comment:
Java coding style discourage using boxed primitive types, unless there are reasons. I think here it's better to just use `boolean`. Same below.

Reply:
Done